tests/CMakeLists.txt

```
@@ -201,7 +201,7 @@ MACRO( ADD_LCIO_TEST file )
     ADD_EXECUTABLE( ${file} EXCLUDE_FROM_ALL ../src/cpp/src/TESTS/${file}.cc )
   ENDIF()
   ADD_DEPENDENCIES( tests ${file} )
-  TARGET_LINK_LIBRARIES( ${file} lcio )
+  TARGET_LINK_LIBRARIES( ${file} lcio sio)
```

```suggestion
  TARGET_LINK_LIBRARIES( ${file} LCIO::lcio SIO::sio)
```

At least the `SIO::sio` one is necessary to fix the linker errors in CI, I think. The lcio target I would just use for consistency and to set the right example.